streamly.cabal

```diff
@@ -102,11 +102,11 @@ extra-source-files:
     design/*.md
     design/*.png
     design/*.rst
-    docs/Overview.md
-    docs/Build.md
-    docs/streamly-vs-async.md
-    docs/streamly-vs-lists.md
-    docs/transformers.md
+    docs/*.md
+    docs/Setup.hs
+    docs/Streamly/Doc/Tutorial.hs
+    docs/doctests.hs
+    docs/streamly-docs.cabal
```

Comment:
This is going to be very tiresome. You'll have 100 documents in the near future in some hirerchy.
`docs/Streamly/Doc/Tutorial/Part1.md`

Also, The module structure generates a hierarchy in the documentation. Do you want `Streamly/Doc/` in that hierarchy? 
Don't think in terms of modules.
- Drop the `Streamly.Doc` prefix.
- Use `Tutorial.hs` (or `Tutorial/Server.hs`) for the tutorial.
Mix in `md` (`rst`) and add them in `extra-doc-files` in `streamly-doc.cabal`

Reply:
The prefix was only to be consistent with tests and benchmarks, but we do not have to be. It can be dropped. We are not going to import these modules or expose them to users so the module names do not matter.